app/components/bali/progress/component.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= tag.div(**options) do %>
+  <%= tag.progress(value: value, max: 100, style: @bar_color) %>
+
+  <% if percentage %>
+    <%= tag.span do %>
```

Comment:
I dont think so we still need this span, this can be the `tag.small` so maybe is good option to use `tag.small`